src/seedu/addressbook/AddressBook.java

```diff
@@ -546,7 +546,7 @@ private static boolean isDeletePersonArgsValid(String rawArgs) {
      * @param rawArgs raw command args string for the delete person command
      * @return extracted index
      */
-    private static int extractTargetIndexFromDeletePersonArgs(String rawArgs) {
+    private static int getDeletePersonArgsIndex(String rawArgs) {
```

Comment:
good thought process here to simplify the method name. while this is not a norm,note that, in Java, generally we use `get*` method for variable getters.